doc/index.rst

```diff
@@ -5,6 +5,29 @@
 Ceph uniquely delivers **object, block, and file storage in one unified
 system**.
 
+Quick Links
+===========
+
+.. image:: images/button_octo_install.png
```

Comment:
the words on the image don't match the text - might look better with a shorter title in the image, e.g. 'Install Guide'